app/models/physical_ticket.rb

```diff
@@ -0,0 +1,9 @@
+class PhysicalTicket < ActiveRecord::Base
+  belongs_to :ticket_purchase
+  has_one :ticket, through: :ticket_purchase
+  has_one :conference, through: :ticket_purchase
+  has_one :user, through: :ticket_purchase
+
+  scope :by_conference, ->(conference) { joins(:ticket_purchase).where('ticket_purchases.conference_id = ?', conference.id) }
```

Comment:
Less dangerous to do the joins, and at the same time less performance. But adding a relationship directly to the conference can end in having wrong data because the duplication.